www/js/app.js

```diff
@@ -69,3 +69,43 @@ angular.module('emission', ['ionic','ionic.service.core', 'emission.controllers'
   $urlRouterProvider.otherwise('/splash');
   console.log("Ending config");
 });
+
+
+function parseURL(url) {
+  var addr = url.split('//')[1];
+  var route = addr.split('?')[0];
+  var params = addr.split('?')[1];
+  var paramsList = params.split('&');
+  var rtn = {route: route};
+  for (var i = 0; i < paramsList.length; i++) {
+    rtn[paramsList[i].split('=')[0]] = paramsList[i].split('=')[1];
+  }
+  return rtn;
+}
+function handleOpenURL(url) {
+  setTimeout(function() {
+    var kvList = parseURL(url);
```

Comment:
Please add a new alert here that shows the parsed components (addr, route, paramsList)